src/reflect/strconv.go

```diff
@@ -0,0 +1,248 @@
+// Copyright 2009 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package reflect
+
+import (
+	"unicode/utf8"
+)
+
+// ErrSyntax indicates that a value does not have the right syntax for the target type.
+var ErrSyntax = badSyntax{}
```

Comment:
This file adds a number of exported symbols. Because this is the reflect package, these (copied) symbols should not be exported.